ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/ConnectExecutor.java

```diff
@@ -43,7 +43,7 @@ public static Optional<KsqlEntity> execute(
   ) {
     final CreateConnector createConnector = statement.getStatement();
     final ConnectClient client = serviceContext.getConnectClient();
-    final String maskedStatement = QueryMask.getMaskedStatement(statement.getStatementText());
+    final String maskedStatement = QueryMask.getMaskedStatement(statement.getMaskedStatementText());
```

Comment:
Do we need to call `getMaskedStatement` again if we already call `getMaskedStatementText`?